src/QAT_RPC/qat_rpc/zmq/wrappers.py

```diff
@@ -68,6 +75,8 @@ def address(self):
 
     def run(self):
         self._running = True
+        with self._metric.receiver_status() as metric:
```

Comment:
OK. The way this is implemented, you'd get this metric only at start up.

Reply:
I'm assuming since this is similar to the way  _running=true is setup, it will remain 1 during the server is up and will go to 0.0 when server stopped. in any case the metric endpoint will be down if receiver itself is not running, so we ideally only expect 1.0 reported whenever receiver and ZMQ are running.